test.py

```
@@ -1,8 +1,5 @@
-import connect
-import lobby
+def x():
+    pass
 
-c = connect.Connection()
 
-c.populate_champ_table()
-
-lobby.create_lobby(c, "draft")
+print(getattr(x, "name"))
```

Ruff (0.15.9)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)